lib/msf/ui/console/module_action_commands.rb

```diff
@@ -29,24 +29,42 @@ def commands
 
   #
   # Allow modules to define their own commands
+  # Note: A change to this method will most likely require a corresponding change to respond_to_missing?
   #
   def method_missing(meth, *args)
-    if (mod and mod.respond_to?(meth.to_s, true) )
+    if mod && mod.respond_to?(meth.to_s, true)
 
       # Initialize user interaction
       mod.init_ui(driver.input, driver.output)
 
       return mod.send(meth.to_s, *args)
     end
 
-    action = meth.to_s.delete_prefix('cmd_')
+    action = meth.to_s.delete_prefix('cmd_').delete_suffix('_tabs')
     if mod && mod.kind_of?(Msf::Module::HasActions) && mod.actions.map(&:name).any? { |a| a.casecmp?(action) }
+      return cmd_run_tabs(*args) if meth.end_with?('_tabs')
       return do_action(action, *args)
     end
 
     super
   end
 
+  #
+  # Note: A change to this method will most likely require a corresponding change to method_missing
+  #
+  def respond_to_missing?(meth, _include_private = true)
```

Comment:
Context: https://thoughtbot.com/blog/always-define-respond-to-missing-when-overriding

And the corresponding calling logic which assumed `respond_to?` was implemented correctly:
https://github.com/rapid7/metasploit-framework/blob/c4639fa626cf04e24d7a2cfd5a9bcc66fdca04d2/lib/rex/ui/text/dispatcher_shell.rb#L467-L471